### PR TITLE
Fix filter quotes wraped insert completion options

### LINKED ISSUE #31425: Autocompletion problems with the new list
**Godot version:**
afecc0b

**Issue description:**
I can't believe no one reported it yet, but the autocompletion has a severe issue since the changes from #29626. Check this
image
It's a very nice list without quotes and with icons looks cool and stuff. But how do I search that list? I mean, when I start typing something to filter options, the list disappears. Previously, all options started with ", so you had to write " and then whatever you wanted to search, otherwise you ran into #24077, which seems to still be true. Now when you write ", the list disappears too, so it's entirely impossible to filter it.

### PR DESCRIPTION
**Pull request by Geequlim**
Do not query new competion when filtering file pathes, node pathes and signals
Fix #31425

### CODE REVIEW

**Comment by KoBeWi** [comment]
Isn't it possible to fix the issue without reintroducing quotes?

**Comment by Geequlim** [comment]
@KoBeWi It does not reintroducing quotes.

**Comment by akien-mga** [comment]
Needs a rebase, otherwise seems fine to merge.

File: scene/gui/text_edit.cpp [hunk 1]
```diff
@@ -6126,10 +6126,30 @@ void TextEdit::query_code_comple() {
...
 	}
 
-	if (ofs > 0 && (inquote || _is_completable(l[ofs - 1]) || completion_prefixes.has(String::chr(l[ofs - 1]))))
-		emit_signal("request_completion");
-	else if (ofs > 1 && l[ofs - 1] == ' ' && completion_prefixes.has(String::chr(l[ofs - 2]))) //make it work with a space too, it's good enough
-		emit_signal("request_completion");
+	bool ignored = completion_active && !completion_options.empty();
+	if (ignored) {
+		ScriptCodeCompletionOption::Kind kind = ScriptCodeCompletionOption::KIND_PLAIN_TEXT;
+		const ScriptCodeCompletionOption *prevois_option = NULL;
```
Comment (bojidar-bg) [inline scene/gui/text_edit.cpp]: Typo: "prevIOUs"

**Comment by Geequlim** [comment]
@akien-mga  Rebased.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**